From d870d54ab3d048088a97f3e159cf4438de884cc5 Mon Sep 17 00:00:00 2001
From: Karl Dietz <dekarl@mythtv.org>
Date: Thu, 9 Jan 2014 01:00:19 +0100
Subject: [PATCH 52/57] strip whitespace when splitting RTSP OPTIONS

Patch by Nicolas Niclausse
Refs #11949

(cherry picked from commit 2131558d64e269cb0b700f4faf126e4fca9b2d47)
---
 mythtv/libs/libmythtv/recorders/cetonrtsp.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/recorders/cetonrtsp.cpp b/mythtv/libs/libmythtv/recorders/cetonrtsp.cpp
index 9040787..0182893 100644
--- a/mythtv/libs/libmythtv/recorders/cetonrtsp.cpp
+++ b/mythtv/libs/libmythtv/recorders/cetonrtsp.cpp
@@ -162,7 +162,7 @@ bool CetonRTSP::GetOptions(QStringList &options)
 {
     if (ProcessRequest("OPTIONS"))
     {
-        options = _responseHeaders.value("Public").split(",");
+        options = _responseHeaders.value("Public").split(QRegExp(",\\s*"));
         return true;
     }
     return false;
-- 
1.7.10.2

